lib/web_ui/test/common/test_initialization.dart

```diff
@@ -22,11 +22,11 @@ void setUpUnitTests({
     }
 
     // Some of our tests rely on color emoji
-    final engine.FlutterConfiguration config = engine.FlutterConfiguration()
-      ..setUserConfiguration(
-        js_util.jsify(<String, Object?>{
-          'useColorEmoji': true,
-        }) as engine.JsFlutterConfiguration);
+    final engine.FlutterConfiguration config = engine.configuration.merge(
+      js_util.jsify(<String, Object?>{
+        'useColorEmoji': true,
```

Comment:
I think it'd been much easier to pass `useColorEmoji: true` on the `window.flutterConfiguration` JS object of the `test_platform.dart`?

That way the value would be read as part of the default egine configuration (since it seems that it's being applied to all tests anyway?)

I can try a PR later if you want!